src/pages/Facility/settings/locations/LocationList.tsx

```diff
@@ -53,6 +58,41 @@ export default function LocationList({ facilityId }: Props) {
     setSelectedLocation(location);
     setIsSheetOpen(true);
   };
+  const { mutate: removeLocation } = useMutation({
+    mutationFn: ({
+      facilityId,
+      locationId,
+    }: {
+      facilityId: string;
+      locationId: string;
+    }) =>
+      mutate(locationApi.delete, {
+        pathParams: { facility_id: facilityId, id: locationId },
+      })({ facilityId, locationId }),
```

Comment:
refer https://docs.ohc.network/docs/care/development/data-fetching-in-care/

let's pass the callback directly instead.